src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -7,7 +7,7 @@
     <changeSet author="Maxim" id="1">
```

Comment:
Best practice for liquibase is split changes to files and have one master files

You can read more here http://www.liquibase.org/bestpractices.html